res/database/Default/database_programmes.xml

```diff
@@ -11514,10 +11512,9012 @@ Dream scenes and sentiments warp the storyline.</en>
+			</description>
+			<staff>
+				<member index="0" function="1">21cf3fe5-98dc-48a0-b84f-a31c54ba2861</member> <!-- TODO: function="5" -->
+				<member index="1" function="2">c39a70f6-b8fd-40fe-96cd-9dcb0e460e53</member>
+				<member index="2" function="2">506bc383-259e-4a36-9278-081aafc8f35c</member>
+				<member index="3" function="2">6e7e605a-597f-421e-aacd-81acc26cf55e</member>
+			</staff>
+			<groups target_groups="0" pro_pressure_groups="0" />
+			<data country="USA" distribution="1" maingenre="17" subgenre="12,14" flags="64" blocks="2" price_mod="1" />
+			<releaseTime year="1987" day="3" />
+			<ratings critics="71" speed="76" outcome="31" />
+		</programme>
+		<programme id="e5d747b5-ac85-4b63-b90b-190d9a4d0aae" product="1" licence_type="1" creator="9551" imdb_id="tt0102070">
+			<title>
+				<de>Hawkson Huds - Der Diebmeister</de>
+				<en>Hawkson Huds</en>
+			</title>
+			<title_original>
+				<de>Hudson Hawk - Der Meisterdieb</de>
+				<en>Hudson Hawk</en>
+			</title_original>
+			<description>
+				<de>[1|Full] in einer abgedrehten Geschichte um die Geheimnisse Leonardo da Vincis. Ein Dieb, frisch aus dem Gefängnis entlassen, will endlich ein normales Leben führen. Aber es kommt natürlich anders.</de>
+				<en>[1|Full] in a twisted tale about Leonardo da Vinci's secrets. A thief, freshly released from prison, at last wants to live a normal life. But things turn out differently, of course.</en>
+			</description>
+			<staff>
+				<member index="0" function="1">b75044b5-2e12-4131-836b-0a977181b376</member>
+				<member index="1" function="2">2b6af8c4-7775-44da-820e-1ce9c55d8cd9</member>
+				<member index="2" function="2">30e987f6-5bf5-43de-99a9-0a3ab667ee8e</member>
+				<member index="3" function="2">c56577b6-79ce-48f7-a494-8fdcddc81b85</member>
+			</staff>
+			<groups target_groups="0" pro_pressure_groups="0" />
+			<data country="USA" distribution="1" maingenre="2" subgenre="5,1" flags="0" blocks="2" price_mod="1" />
+			<releaseTime year="1991" day="5" />
+			<ratings critics="34" speed="54" outcome="26" />
+		</programme>
+		<programme id="a214650c-f41e-4e43-b57e-7c53e0753814" product="1" licence_type="1" creator="9551" imdb_id="tt0088855">
+			<title>
+				<de>Turn on the Night</de>
+				<en>Turn on the Night</en>
+			</title>
+			<title_original>
+				<de>Bring on the Night</de>
+				<en>Bring on the Night</en>
+			</title_original>
+			<description>
+				<de>Konzertfilm von und mit [0|Full], dem britischen Musiker und Schauspieler. Dokumentiert die Vorbereitungen und Proben für seinen ersten Soloauftritt. Seine Begleitband kommt ebenfalls zu Wort.</de>
+				<en>Concert film by and with the British musician and actor [0|Full]. Documents the preparation and rehearsals for his first solo show. His backing band also gets a word in edgewise.</en>
+			</description>
+			<staff>
+				<member index="0" function="2">687adb8f-943b-41d6-bfdb-7e49a305774f</member> <!-- TODO: function="18" -->
```

Comment:
Note that not all jobs are "cast jobs", so not all jobs would be listed

Reply:
that's fine with me.